examples/subscription.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script>
```

Comment:
Perhaps, placing the script tag at the end of the body tag is a good choice to avoid script functionality errors when loading the html element.